R/PipeOpFFS.R

```diff
@@ -57,10 +59,40 @@ PipeOpFFS = R6Class("PipeOpFFS",
         drop = p_lgl(tags = c("train", "predict", "required")),
         left = p_dbl(tags = c("train", "predict", "required")),
         right = p_dbl(tags = c("train", "predict", "required")),
-        feature = p_fct(
-          levels = c("mean", "max", "min", "slope", "median", "var"),
-          tags = c("train", "predict", "required")
-        )
+        features = p_uty(tags = c("train", "predict", "required"), custom_check = function(x) {
+          if (!(is.character(x) || is.list(x))) {
+            return("Features must be a character or list")
+          }
+          if (is.character(x)) {
+            return(check_subset(x, choices = c("mean", "median", "min", "max", "slope", "var")))
+          }
+          res = check_list(x, types = c("character", "function"), any.missing = FALSE, unique = TRUE)
+          if (!isTRUE(res)) {
+            return(res)
+          }
+          nms = names2(x)
```

Comment:
you can use `checkmate::test_names()` for that

Reply:
I haven't used `checkmate::test_names` since it returns logical instead of the error message as a character

Reply:
then you can use `check_names()`